src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/CalendarView/CalendarView_Theming.xaml.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Uno.UI.Samples.Controls;
-using Windows.UI;
-using Windows.UI.ViewManagement;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
+using Uno.Helpers.Theming;
+#if !WINAPPSDK
+using Uno.UI.Samples.Controls;
+#endif
```

Comment:
Uno.UI.Samples.Controls for [sample-stuff] attributes used on any-plat
Uno.Helpers.Theming for the utility method used only in uno-plats aka non-windows
```suggestion
using Uno.UI.Samples.Controls;
#if !WINAPPSDK
using Uno.Helpers.Theming;
#endif
```